app/localization/translated/uk.json

```diff
@@ -851,6 +847,10 @@
   "Footer.build": "Збірка",
   "Footer.contact": "Зв’яжіться з нами",
   "Footer.copyright": "Всі права захищені",
+  "Footer.deleteImportantLaunch": "Delete Important Launch",
+  "Footer.deleteImportantLaunches": "Delete Important Launches",
+  "Footer.deleteOnlyRegular": "Delete only regular",
+  "Footer.deleteWithImportantLaunch": "Delete with Important Launches",
```

Comment:
It looks like the 'manage:translations' script was accidentally not run for these messages
![image](https://github.com/reportportal/service-ui/assets/31861805/a6a98eb3-c7e1-4727-9ae2-bbe6fa5df953)
So if it is changed in the scope of this PR, could you please use the deleted translations for these messages? The same ask about other languages.